packages/@sanity/cli/package.json

```
@@ -62,6 +62,8 @@
   },
   "dependencies": {
     "@babel/traverse": "^7.19.0",
+    "@vercel/frameworks": "1.3.2",
+    "@vercel/fs-detectors": "3.8.4",
```

These packages now have the correct license fields, and I also landed a PR to update a dependency that was causing snyk to report a ReDoS vuln, so we should update:

```suggestion
    "@vercel/frameworks": "1.3.4",
    "@vercel/fs-detectors": "3.8.9",
```